### PR TITLE
Fix API Errors

### PR DESCRIPTION
**Pull request by jefmoura**
## Purpose
Fix API errors and create unit tests for them.
Related issues: #738, #736, #733, #732 

### Considerations
- A new field organization was added in some of the models, so the migration scripts have to be executed
- The new fields are using a default org with id 0

### CODE REVIEW

File: indicators/models.py [hunk 1]
```diff
@@ -75,6 +75,7 @@ class StrategicObjective(models.Model):
     description = models.TextField(max_length=765, blank=True)
     create_date = models.DateTimeField(null=True, blank=True)
     edit_date = models.DateTimeField(null=True, blank=True)
+    organization = models.ForeignKey(Organization, default=0)
```
Comment (Menda) [inline indicators/models.py]: So better blank and null :D

File: feed/serializers.py [hunk 2]
```diff
@@ -305,6 +305,15 @@ class Meta:
 class StrategicObjectiveSerializer(serializers.HyperlinkedModelSerializer):
     id = serializers.ReadOnlyField()
 
+    def create(self, validated_data, **kwargs):
+        user = self.context['request'].user
+        if not user.is_superuser:
```
Comment (Menda) [inline feed/serializers.py]: So yeah, delete this check

File: feed/serializers.py [hunk 3]
```diff
@@ -490,6 +499,15 @@ class Meta:
 class IssueRegisterSerializer(serializers.HyperlinkedModelSerializer):
     id = serializers.ReadOnlyField()
 
+    def create(self, validated_data, **kwargs):
+        user = self.context['request'].user
+        if not user.is_superuser:
```
Comment (Menda) [inline feed/serializers.py]: And this

File: feed/views.py [hunk 4]
```diff
@@ -755,7 +755,7 @@ def list(self, request):
             queryset = Organization.objects.all()
         else:
             user_org = TolaUser.objects.get(user=request.user).organization
-            queryset = Organization.objects.all().filter(organization=user_org)
+            queryset = Organization.objects.all().filter(id=user_org.id)
```
Comment (Menda) [inline feed/views.py]: I think that you don't need the all() if you use filter()

File: feed/views.py [hunk 5]
```diff
@@ -498,12 +498,12 @@ def list(self, request):
         if request.user.is_superuser:
             queryset = StrategicObjective.objects.all()
         else:
-            user_org = TolaUser.objects.get(user=request.user).organization
-            queryset = StrategicObjective.objects.all().filter(country__organization=user_org)
+            user = TolaUser.objects.get(user=request.user)
+            queryset = StrategicObjective.objects.all().filter(country=user.country, organization=user.organization)
```
Comment (Menda) [inline feed/views.py]: I think that you don't need the all() if you use filter()

Comment (jefmoura) [reply feed/views.py]: You're right. We don't need to use the all() but we're using in many other places. Maybe, we should change them in the future as well.

**Pull request merged**